modin/core/execution/ray/implementations/pandas_on_ray/partitioning/virtual_partition.py

```diff
@@ -111,13 +111,12 @@ def list_of_blocks(self):
         List
             A list of ``ray.ObjectRef``.
         """
-        # Defer draining call queue until we get the partitions
-        # TODO Look into draining call queue at the same time as the task
-        result = [None] * len(self.list_of_partitions_to_combine)
-        for idx, partition in enumerate(self.list_of_partitions_to_combine):
-            partition.drain_call_queue()
-            result[idx] = partition.oid
-        return result
+        # Draining call queue of partition is happening during access to physical
+        # data of partition. Defer draining call queue until we get the partitions.
+        # TODO: Look into draining call queue at the same time as the task.
```

Comment:
@devin-petersohn I think you wrote this TODO. How would we do that? Should we file an issue for it?

Reply:
Thread is out of date. Therefore I resolve conversation. The comment was reverted

PR scope was changed to cover only 1st problem described in https://github.com/modin-project/modin/issues/4530 (renaming oid/future/data -> _data). The public getter will be added in an other PR as resolution of 2nd and 3rd problems from https://github.com/modin-project/modin/issues/4530.